tests/behaviour/behaviour.test.ts

```diff
@@ -33,16 +33,18 @@ interface AsyncTestCluster {
   dependencies: Map<string, string[]>;
 }
 
-const accountDir = '.starknet_accounts_devnet';
-fs.mkdirSync(accountDir, { recursive: true });
```

Comment:
Is this not needed anymore?

Reply:
Yes, the location is `tests/cli` so until someone deletes the directory from repository it's not needed